ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,365 @@
 
                                                                    # ]]]
-# .. envvar:: phpipam_domain [[[
+# .. envvar:: phpipam__nginx_auth_realm [[[
 #
-# Domain on which phpIPAM is configured in nginx
-phpipam_domain: [ 'ipam.{{ ansible_domain }}' ]
+# Webserver authentication realm.
+phpipam__nginx_auth_realm: 'IPAM access is restricted'
 
                                                                    # ]]]
-# .. envvar:: phpipam_nginx_auth_realm [[[
+# .. envvar:: phpipam__nginx_access_policy [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_nginx_auth_realm: 'IPAM access is restricted'
+# Name of the "nginx access policy" for phpIPAM. See ``debops.nginx``
+# Ansible role for more details.
+phpipam__nginx_access_policy: ''
 
                                                                    # ]]]
-# .. envvar:: phpipam_nginx_access_policy [[[
+# .. envvar:: phpipam__webserver_user [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_nginx_access_policy: ''
+# phpIPAM webserver user (needs read-only access to the website code)
+phpipam__webserver_user: '{{ ansible_local.nginx.user
+                             if (ansible_local is defined and
+                                 ansible_local.nginx is defined and
+                                 ansible_local.nginx.user is defined)
+                             else "www-data" }}'
 
                                                                    # ]]]
-# ---- User, group, database configuration ----
+# ]]]
 
-# .. envvar:: phpipam_user [[[
+# UNIX environment [[[
+# --------------------
+
+# .. envvar:: phpipam__user [[[
 #
-# System user and group for phpIPAM application
-phpipam_user: 'phpipam'
+# System user for phpIPAM application
+phpipam__user: 'phpipam'
 
                                                                    # ]]]
-# .. envvar:: phpipam_group [[[
+# .. envvar:: phpipam__group [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_group: 'phpipam'
+# System group for phpIPAM application
+phpipam__group: 'phpipam'
 
                                                                    # ]]]
-# .. envvar:: phpipam_home [[[
+# .. envvar:: phpipam__home [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_home: '{{ (ansible_local.root.home
+# Home directory of the ``phpipam__user`` user.
+phpipam__home: '{{ (ansible_local.root.home
                    if (ansible_local|d() and ansible_local.root|d() and
                        ansible_local.root.home|d())
-                   else "/var/local") + "/" + phpipam_user }}'
+                   else "/var/local") + "/" + phpipam__user }}'
 
                                                                    # ]]]
-# .. envvar:: phpipam_database_user [[[
+                                                                   # ]]]
+# Database configuration [[[
+# --------------------------
+# .. envvar:: phpipam__database_user [[[
```

Comment:
There should be an empty line between section title and first variable, otherwise Sphinx rendering might break.

Reply:
Done